bluesky_queueserver/profile_collection_sim/allowed_plans_and_devices.yaml

```diff
@@ -0,0 +1,56 @@
+allowed_devices:
+- motor
+- motor1
+- motor2
+- motor3
+- jittery_motor1
+- jittery_motor2
+- noisy_det
+- det
+- identical_det
+- det1
+- det2
+- det3
+- det4
+- det5
+- ab_det
+- direct_img
+- direct_img_list
+- pseudo3x3
+- pseudo1x3
+- det_with_conf
+- det_with_count_time
+- motor_no_pos
+- motor_empty_hints1
+- motor_empty_hints2
+- motor_no_hints1
+- motor_no_hints2
+allowed_plans:
+- count
+- list_scan
+- rel_list_scan
+- list_grid_scan
+- rel_list_grid_scan
+- log_scan
+- rel_log_scan
+- adaptive_scan
+- rel_adaptive_scan
+- tune_centroid
+- scan_nd
+- inner_product_scan
+- scan
+- grid_scan
+- rel_grid_scan
+- relative_inner_product_scan
+- rel_scan
+- tweak
+- spiral_fermat
+- rel_spiral_fermat
+- spiral
+- rel_spiral
+- spiral_square
+- rel_spiral_square
+- ramp_plan
+- fly
+- x2x_scan
+- move_then_count
```

Comment:
Was this intentionally committed? I guess for real `profile_collection`s we are going to have similar YAML configurations for the allowed devices/plans. I think it's a really good way to limit what can be used via the queueserver/API.

Reply:
Yes, it was intentionally committed. There is a function that creates complete list which is supposed to be run only once. Then the list can be edited manually. I guess the idea is to keep separate lists for each group of users in the database. But I think that it is useful to have capability of working just with profile collection.